lib/rules/security/func-visibility.js

```diff
@@ -61,18 +63,44 @@ class FuncVisibilityChecker extends BaseChecker {
       config && config.getObjectPropertyBoolean(ruleId, 'ignoreConstructors', false)
   }
 
+  SourceUnit(node) {
```

Comment:
A better way to do this is to set a variable when a contract definition is entered, and only apply the rule in those cases. See [this](https://github.com/protofire/solhint/blob/129251621156093557f4aeedba240bb5182f1733/lib/rules/miscellaneous/comprehensive-interface.js#L39-L45) for an example of how to do that.

Reply:
done